javascript-algorithms-and-data-structures/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/index.html

```diff
@@ -1,140 +1,106 @@
-          </li>
-          <li>
-            If you choose a selected score, then you will move to the next round
-          </li>
-          <li>
-            If you decline to choose a selected score, then you can roll the
-            dice again two more times
-          </li>
-        </ul>
-        <h2 class="points">Points</h2>
-        <ul>
-          <li>Three of a kind: Sum of all five dice</li>
-          <li>Four of a kind: Sum of all five dice</li>
-          <li>Full house: Three of a kind and a pair - 25 points</li>
-          <li>
-            Small straight: Four of the dice have consecutive values - 30 points
-          </li>
-          <li>
-            Large straight: All five dice have consecutive values - 40 points
-          </li>
-        </ul>
-      </div>
-    </header>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Advanced Dice Game</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
 
-    <main>
-      <form id="game">
-        <div id="dice">
-          <div class="die"></div>
-          <div class="die"></div>
-          <div class="die"></div>
-          <div class="die"></div>
-          <div class="die"></div>
-        </div>
+<body>
+  <header>
+    <h1>Advanced Dice Game</h1>
+    <button class="btn" id="rules-btn" type="button">Show rules</button>
+    <div class="rules-container">
+      <h2>Rules</h2>
+      <ul>
+        <li>There are total of six rounds</li>
+        <li>You can only roll the dice three times per round</li>
+        <li>To start the game, roll the dice</li>
+        <li>
+          Then, choose from one of the selected scores or roll the dice again
+        </li>
+        <li>
+          If you choose a selected score, then you will move to the next round
+        </li>
+        <li>
+          If you decline to choose a selected score, then you can roll the
+          dice again two more times
+        </li>
+      </ul>
+      <h2 class="points">Points</h2>
+      <ul>
+        <li>Three of a kind: Sum of all five dice</li>
+        <li>Four of a kind: Sum of all five dice</li>
+        <li>Full house: Three of a kind and a pair - 25 points</li>
+        <li>
+          Small straight: Four of the dice have consecutive values - 30 points
+        </li>
+        <li>
+          Large straight: All five dice have consecutive values - 40 points
+        </li>
+      </ul>
+    </div>
+  </header>
 
-        <p class="rounds-text">
-          <strong>Rolls:</strong> <span id="current-round-rolls">0</span> |
-          <strong>Round:</strong> <span id="current-round">1</span>
-        </p>
+  <main>
+    <form id="game">
+      <div id="dice">
+        <button class="die" type="button"></button>
+        <button class="die" type="button"></button>
+        <button class="die" type="button"></button>
+        <button class="die" type="button"></button>
+        <button class="die" type="button"></button>
```

Comment:
We shouldn't use buttons here if they're not interactable.

Reply:
changes made @naomi-lgbt